codemods/jscodeshift/immutable/4/isterable-to-iscollection/index.js

```diff
@@ -0,0 +1,42 @@
+/*
+The MIT License (MIT)
+
+Copyright (c) 2023 Rajasegar Chandran
```

Comment:
I think you are the creator of this codemod right?

Reply:
resolved in #25

Reply:
resolved in [removed redundant license](https://github.com/intuita-inc/codemod-registry/pull/25/commits/8bbe0b442a9f5ac1521c698a9287e2f7116c8223)